plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -15,25 +15,29 @@
 // limitations under the License.
 //
 
-proc mayaUsdTranslatorExport_SetCheckbox(string $arg, string $widget) {
+proc mayaUsdTranslatorExport_SetCheckbox(string $arg, int $enable, string $widget) {

```

Comment:
All the setters take an extra argument to enable the field or not depending if the value was set from a regular argument, or from an argument provided by a job context.